### ISSUE TITLE
AttributeError: 'IPythonDisplay' object has no attribute 'to_string'

### ISSUE THREAD

**Issue by WannabeKid** [pydicom/deid#100]
Can anyone help me on this please ?
I was trying to display the largest vale of a column using nlargest method

**Comment by WannabeKid** [comment]
I was trying to find the largest value of a column of a dataset and then display it.

display(nobel.nlargest(1, "age"))


this is the line of code i used.
nobel is the dataframe name


I am a newbie to this field and trying to learn

**Comment by vsoch** [comment]
Are you using pandas dataframes (hence the reference to columns?) If you have a column, let's say it's named "age", you should be able to do:

```python
nobel['age'].max()
```
You shouldn't need to use any sort of display().

**Comment by vsoch** [comment]
Also, if you are having (general trouble) or questions with python, dataframes, it's probably best to try stack overflow or a Github board relevant to the software you are using. pydicom/deid is a python module for working with dicom datasets, specifically headers.

**Comment by WannabeKid** [comment]
I was doing a project on datacamp where i came across this. here is the link to the project

https://projects.datacamp.com/projects/441

And thanks for the advice vsoch, means a lot :-)

**Comment by MAStarsis** [comment]
Hello! I did the same project on Datacamp, and for some reason, it worked removing display, not using anything. Probably is an error that they have there.

**Comment by vsoch** [comment]
Awesome! Thanks for sharing - others that find the issue here will know to just leave it out.

**Comment by RachelJau** [comment]
I came across the same problem when I was doing the DataCamp project. And the solution is to remove all the print(), display() function. I think they just set the format of answers they want to get(*_*)

**Comment by vsoch** [comment]
Why is this error related to deid?

**Comment by amaniBoughanmi** [comment]
I have the same issue on the same Datacamp task, how did you resolve it please ?

**Comment by aladushka** [comment]
I removed display() as advised by MAStarsis and it worked.

**Comment by tanyabaramzina** [comment]
Thanx! Had the very same problem with DataCamp task and couldn't figure out what's wrong. Removing display() helped.

**Comment by ayepremyan** [comment]
Actually, you just need to keep only the first display() and remove the second. I think the aim of this task was to show how the display() function is working, but the question is formulated in somewhat tricky way. I think they wanted the students to repeat the same task as they did in question 2.

**Comment by surajghimire** [comment]
Found this problem on Datacamp. Sad to see they have such silly errors even after two years.

**Comment by vsoch** [comment]
It’s also very funny that the issue reported and discussed here has nothing to do with the library at hand!

**Comment by MarcinKamil84** [comment]
Like @ayepremyan said: it's not an issue BUT:

* if you code

`# The oldest winner of a Nobel Prize as of 2016
display(nobel.nlargest(1, 'age'))

# The youngest winner of a Nobel Prize as of 2016
nobel.nsmallest(1, 'age')`

all works fine.

However, if you try this:

`# The oldest winner of a Nobel Prize as of 2016
display(nobel['age'].nlargest(1)

# The youngest winner of a Nobel Prize as of 2016
nobel['age'].nsmallest(1)`

you will be getting the error message.

**Comment by kchoward48** [comment]
> Like @ayepremyan said: it's not an issue BUT:
> 
> * if you code
> 
> ```
> display(nobel.nlargest(1, 'age'))
> nobel.nsmallest(1, 'age')
> ```
> 
> all works fine.
> 
> However, if you try this:
> 
> ```
> display(nobel['age'].nlargest(1)
> nobel['age'].nsmallest(1)
> ```
> 
> you will be getting the error message.

This is correct. I worked on the same project. As for the reason, it is a default behavior in Jupyter Notebook: 

'By default, a Jupyter Notebook (which is where you are working right now) will only show the final output in a cell. If you want to show intermediate results, you will have to use the display() function.'

So use the former code snippet in the quoted part of this post.
